<launch>

    <!-- Specifica il pacchetto che contiene il nodo ROS -->
    <arg name="package" default="panda_controllers" />
    <arg name="node_name" default="trajectory_visualizer" />

    <!-- Specifica il percorso del file YAML contenente i parametri -->
    <arg name="yaml_file" default="$(find panda_controllers)/config/panda_trajectory.yaml" />
    <rosparam command="load" file="$(arg yaml_file)" />

    <node name="$(arg node_name)" pkg="$(arg package)" type="trajectory_visualizer" output="screen"/>

</launch>
